### PR TITLE
add scripts to output k-mers and k-mer counts

### PR DESCRIPTION
**Pull request by ctb**
An initial implementation of a simple script to output k-mer counts.

Per request by @drtamermansour.

### CODE REVIEW

**Comment by mr-c** [comment]
Sweet. This is a great foundation for implementing #724

**Comment by ctb** [comment]
- [ ] write a recipe for both scripts
- [ ] create an issue to move the final for loop(s) into a C++ function as with abundance-dist.py

**Comment by ctb** [comment]
@mr-c @camillescott @luizirber please review at your leisure.

File: sandbox/count-kmers-single.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    parser.add_argument('input_sequence_filename', help='The input'
+                        ' FAST[AQ] sequence file.')
+
+    parser.add_argument('-o', '--out', metavar="output_file",
+                        dest='output_file',
+                        type=argparse.FileType('w'),
+                        default=None, help='output counts to this file')
+    parser.add_argument('--report-total-kmers', '-t', action='store_true',
+                        help="Prints the total number of k-mers to stderr")
```
Comment (mr-c) [inline sandbox/count-kmers-single.py]: This should just happen without a flag. Flagged version was only done so as to not break the CLI API.

**Comment by mr-c** [comment]
Otherwise, LGTM

File: sandbox/count-kmers-single.py (hunk 1 above)
Comment (ctb) [inline sandbox/count-kmers-single.py]: So, how do you want to handle this in general? Let's touch base on it today and then write up an issue.

Comment (ctb) [inline sandbox/count-kmers-single.py]: (I take your point that this can be default for new scripts :)

**Comment by mr-c** [comment]
Otherwise, LGTM

**Comment by ctb** [comment]
Points in https://github.com/dib-lab/khmer/pull/983#issuecomment-100564626 punted to issues; ready for review and merge @mr-c @luizirber @camillescott

**Comment by mr-c** [comment]
LGTM, done

**Pull request merged**